src/main/resources/data-repository-openapi.yaml

```diff
@@ -976,49 +1003,143 @@ paths:
           schema:
             $ref: '#/definitions/ErrorModel'
         403:
-          description: No permission to remove an asset from a dataset
+          description: No permission to ingest
+          schema:
+            $ref: '#/definitions/ErrorModel'
+        409:
+          description: Someone else is using the load tag
           schema:
             $ref: '#/definitions/ErrorModel'
 
-  '/api/repository/v1/register/user':
+  '/api/repository/v1/datasets/{id}/files/bulk/{loadtag}':
     get:
-      description: |
-        Returns whether the user is registered with terra
-      operationId: user
+      description: |-
+        Retrieve the results of a bulk file load. The results of each bulk load are stored
+        in the dataset. They can be queried directly or retrieved with this paginated
+        interface.
+      operationId: bulkFileResultsGet
       tags:
       - repository
+      consumes:
+      - application/json
+      parameters:
+      - $ref: '#/parameters/Id'
+      - $ref: '#/parameters/LoadTag'
+      - in: query
+        name: jobId
+        type: string
+        description: The job id associated with the load
+      - in: query
+        name: offset
+        type: integer
+        default: 0
+        description: The number of items to skip before starting to collect the result set.
+      - in: query
+        name: limit
+        type: integer
+        description: The numbers of items to return.
+        default: 10
       responses:
+        202:
+          description: Job status of bulk load result retrieval
+          schema:
+            $ref: '#/definitions/JobModel'
+          headers:
+            location:
+              type: string
+              description: url for the job polling
         200:
-          description: User Info
+          description: Redirect for bulk load result retrieval complete
           schema:
-            $ref: '#/definitions/UserStatusInfo'
+            $ref: '#/definitions/JobModel'
+          headers:
+            location:
+              type: string
+              description: url for the job result
+        400:
+          description: Bad request
+          schema:
+            $ref: '#/definitions/ErrorModel'
+        403:
+          description: No permission to access bulk file results
+          schema:
+            $ref: '#/definitions/ErrorModel'
 
-  '/api/repository/v1/datasets/{id}/files':
+    delete:
+      description: |-
+        Delete results from the bulk file load table of the dataset.
+        If jobId is specified, then only the results for the loadTag plus that jobId are
+        deleted. Otherwise, all results associated with the loadTag are deleted.
```

Comment:
loadTag+jobId would give different results than just loadTag if the user called bulk load twice with the same loadTag. For example, the first bulk load completed partially and returned some errors. The user fixed the errors and then resubmitted the bulk load to complete fully. Is that correct?